### PR TITLE
NEWS: start adding 236

### PR DESCRIPTION
**Pull request by grawity**
As asked in #7099. Not sure if I did it right

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,5 +1,20 @@
 systemd System and Service Manager
 
+CHANGES WITH 236:
```
Comment (keszybz) [inline NEWS]: I think we should keep adding " in spe" here, in order to not confuse people.

File: NEWS [hunk 2]
```diff
@@ -1,5 +1,20 @@
...
+
+	* The modprobe.d drop-in introduced in v235 has been extended to also
```
Comment (keszybz) [inline NEWS]: ... introduced in v235 for the bonding module ...

File: NEWS [hunk 3]
```diff
@@ -1,5 +1,20 @@
...
+	  set the dummy module option numdummies=0, resolving issues with the
+	  kernel creating dummy0.
+
+        Contributions from: bleep_blop, Christian Hesse, Daniel Rusek, Florian
```
Comment (keszybz) [inline NEWS]: I think it's better to skip the contributor list now, and just create it at the end.

**Comment by poettering** [comment]
lgtm

**Comment by keszybz** [comment]
I don't think we need to wait for CI for documentation updates. Even for man page updates, we can only check syntax, and semaphore is enough for this. For NEWS and such even semaphore is useless, because there are not checks...

This patch has strange indentation, but let's follow the general approach with NEWS and fix that up later. Doesn't make sense for the PR to go through yet another round.

**Comment by grawity** [comment]
Ah, vim didn't pick up the tab settings for some reason...

CI seems to be forever failing on systemd-timesyncd.

**Comment by keszybz** [comment]
I should be now fixed by #7148.

**Pull request merged**